docs/node/web3.md

```diff
@@ -0,0 +1,201 @@
+    # 00000000000000000000000000000000000000000000000072c8215e60d5bca7 # Emerald on Testnet
+    # 000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c # Sapphire on Testnet
+    "<emerald_paratime_id>":
+      estimate_gas_by_simulating_contracts: true
+      allowed_queries:
+        - all_expensive: true
+    "<sapphire_paratime_id>":
+      estimate_gas_by_simulating_contracts: true
+      allowed_queries:
+        - all_expensive: true
+```
+
+## Download the Web3 Gateway
+
+Check the required version of the Web3 gateway based on which network you will
+be using it with ([Mainnet], [Testnet]). Next, download the Oasis-provided
+binaries from the [official github repository][github-releases].
+
+Alternatively, you can download the source release and compile it yourself.
+Consult the [README] file for more information.
+
+## Running the Gateway on Testnet/Mainnet
+
+Copy the content below to the config file of your gateway and adapt it
+accordingly to fit your needs:
+
+```yaml title='gateway.yml'
+# Uncomment the runtime_id below.
+#runtime_id: "000000000000000000000000000000000000000000000000e2eaa99fc008f87f" # Emerald on Mainnet
+#runtime_id: "00000000000000000000000000000000000000000000000072c8215e60d5bca7" # Emerald on Testnet
+#runtime_id: "000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c" # Sapphire on Testnet
+# Path to your internal.sock file in the root Oasis node datadir.
+node_address: "unix:<path-to-oasis-node-unix-socket>"
+
+# By default, we index the entire blockchain history.
+# If you are low on disk space or you use the gateway just for submitting transactions, enable
+# pruning below.
+enable_pruning: false
+pruning_step: 100000
+indexing_start: 0
+
+log:
+  level: debug
+  format: json
+
+database:
+  # Change host and port, if PostgreSQL is running somewhere else.
+  host: "127.0.0.1"
+  port: 5432
+  # Enter your database name, username and password.
+  db: <postgresql_db>
+  user: <postgresql_user>
+  password: <postgresql_password>
+  dial_timeout: 5
+  read_timeout: 10
+  write_timeout: 5
+  max_open_conns: 0
+
+gateway:
+  # Uncomment the chain_id below.
+  #chain_id: 42262 # Emerald on Mainnet
+  #chain_id: 42261 # Emerald on Testnet
+  #chain_id: 23295 # Sapphire on Testnet
+  http:
+    # Change host to your external IP address if you have users accessing Web3 from the outside.
+    host: "localhost"
+    port: 8545
+    cors: ["*"]
+  ws:
+    # Change host to your external IP address if you have users accessing Web3 from the outside.
+    host: "localhost"
+    port: 8546
+    origins: ["*"]
+  method_limits:
+    get_logs_max_rounds: 100
+```
+
+:::tip
+
+All configuration settings can also be set via environment variables. For
+example, you can set the database password by exporting:
+
+```shell
+DATABASE__PASSWORD=<postgresql_password>
+```
+
+environment variable.
+
+:::
+
+Start the gateway:
```

Comment:
We should emphasize that this command is just for testing. In production environments, a service manager should be set up (like https://docs.oasis.io/node/run-your-node/prerequisites/system-configuration#non-root).